.github/workflows/deploy_rovecomm.yml

```diff
@@ -46,18 +46,14 @@ jobs:
 
       - name: Install Project Dependencies
         run: |
-          sudo apt update && sudo apt install -y libeigen3-dev zip
-          wget -q https://github.com/MissouriMRDT/Autonomy_Packages/raw/main/quill/amd64/quill_${QUILL_VERSION}_amd64.deb
```

Comment:
Is there a problem with Quill?

Reply:
Quill was removed from the project since it was causing issues with using the library in other projects. Since it wasn't linked statically, RoveComm depended on the other project properly including Quill in way that only really worked for Autonomy_Software. Now RoveComm_CPP has no external dependencies except for pthreads and the socket library which should be linked statically now.